tests/valid-test.toit

```diff
@@ -40,19 +45,26 @@ main:
   expect-not (semver.is-valid "1.0.0+ä")
   expect-not (semver.is-valid "1.0.0-a-z.A-Z.0-9.00")
 
-  expect (semver.is-valid "v1.0.0")
-  expect-not (semver.is-valid --no-allow-v "v1.0.0")
+  expect-not (semver.is-valid "v1.0.0")
+  expect (semver.is-valid "v1.0.0" --accept-v)
+
+  // Needed to drop --no-require-major-minor-patch: can't start with "no".
+  // changed to
```

Comment:
I don't understand.
`--no-X` is equivalent to `--X=false`.

Does that mean that we don't have `--require-major-minor-patch` anymore?